packages/flex-plugin-e2e-tests/src/core/parameters.ts

```diff
@@ -91,6 +92,7 @@ export const testParams: TestParams = {
     operatingSystem,
     cwd: process.cwd(),
     ignorePrefix: process.env.NPM_IGNORE_PREFIX === 'true' || false,
+    nodeOptions: semver.gte(process.env.NODE_VERSION as string, '17.0.0') ? '--openssl-legacy-provider' : undefined,
```

Comment:
Is it required from 17 onwards? Does this work seamlessly when e2e is run in local environment in different node versions?

Reply:
Yes, openssl issue is seen in node version >=17. Yes but we would need to set an extra env variable NODE_VERSION with the version we want to run e2e with. Added this in the readme file of this package.